lib/compat/wordpress-6.7/rest-api.php

```diff
@@ -156,3 +156,69 @@ function gutenberg_register_post_type_args_for_wp_global_styles( $args, $post_ty
 }
 
 add_filter( 'register_post_type_args', 'gutenberg_register_post_type_args_for_wp_global_styles', 10, 2 );
+
+if ( ! function_exists( 'register_deactivate_plugins_endpoint' ) ) {
+	/**
+	 * Registers the custom REST API route for deactivating all plugins.
+	 */
+	function register_deactivate_plugins_endpoint() {
```

Comment:
I think this would feel much more at home within the `WP_REST_Plugins_Controller` class. Not sure if this is only required to be here to be implemented in the plugin or not, but [the Core ](https://github.com/WordPress/wordpress-develop/pull/7775) also includes this here.